src/main/scala/com/redislabs/provider/redis/redisFunctions.scala

```diff
@@ -267,6 +283,15 @@ class RedisContext(@transient val sc: SparkContext) extends Serializable {
     kvs.foreachPartition(partition => setHash(hashName, partition, ttl, redisConfig, readWriteConfig))
   }
 
+  /**
+    *
+    * @param kvs RDD of values
+    */
+  def toRedisHLL(kvs: RDD[(String, String)])
+                (implicit redisConfig: RedisConfig = new RedisConfig(new RedisEndpoint(sc.getConf))): Unit = {
```

Comment:
ReadWriteConfig parameter is missing, please see other methods like this e.g. `toRedisZSET`